### PR TITLE
CHE-1094: GDB Debugger Connections UX Impovements

### PR DESCRIPTION
**Pull request by tolusha**
- Added <Debug> button on Debug Configuration Page.
- Added <Debug on workspace machine> check box to simplify GDB configuration to debug locally

@sleshchenko 
@azatsarynnyy 
@vzhukovskii 

pls review

### CODE REVIEW

**Comment by sleshchenko** [comment]
LGTM

File: plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/configuration/EditDebugConfigurationsView.java [hunk 1]
```diff
@@ -97,6 +100,9 @@
         /** Called when 'Cancel' button is clicked. */
         void onCancelClicked();
 
+        /** Called when 'Cancel' button is clicked. */
```
Comment (benoitf) [inline plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/configuration/EditDebugConfigurationsView.java]: it's when debug button is clicked no ?

Comment (tolusha) [inline plugins/plugin-debugger/che-plugin-debugger-ide/src/main/java/org/eclipse/che/plugin/debugger/ide/configuration/EditDebugConfigurationsView.java]: correct

**Comment by azatsarynnyy** [comment]
ok

**Comment by vparfonov** [comment]
ok

**Pull request merged**